playwright/helper.py

```diff
@@ -243,3 +243,14 @@ class RouteHandlerEntry:
     def __init__(self, matcher: URLMatcher, handler: RouteHandler):
         self.matcher = matcher
         self.handler = handler
+
+def raise_not_installed_error(message: str) -> None:
+    raise Exception(f"""
+================================================================================
+{message}
+Please complete Playwright installation via running
+
+    "python -m playwright install"
```

Comment:
If we compare Playwright Python with Pytest, they also use just the `pytest` command instead of `python -m pytest`. So for that I would use playwright install too.

Reply:
Same as above, let's celebrate the discoverable execution once we have cli options.